### ISSUE TITLE
-b option doesn't work

### ISSUE THREAD

**Issue by texttheater** [mjpost/bibsearch#36]
Using the `search` command with the `-b` option.

Expected behavior: shows output in bibtex format.

Actual behavior: shows text summary, same as without the option.

```
$ bibsearch search damonte incremental -b
1. [damonte2017:incremental] Marco Damonte, Shay B. Cohen and Giorgio
   Satta. 2017. "An Incremental Parser for Abstract Meaning
   Representation". Proceedings of the 15th Conference of the European
   Chapter of the Association for Computational Linguistics: Volume 1,
   Long Papers. http://aclweb.org/anthology/E17-1051
[INFO] Download and display any of these PDFs with 'bibsearch open N' (N the index).
```

**Comment by cl3wis** [comment]
I think I have a fix in PR #38

**Comment by davvil** [comment]
Thanks for reporting this and providing a fix so quickly!

At the moment both Matt and I are on vacation. We will revise and merge in
one or two weeks time.

**Comment by mjpost** [comment]
This was fixed in commit 467bcd802e5e439fbb2418f9643311f0ba93e1bf.
